src/canmatrix/cmjson.py

```diff
@@ -101,21 +101,21 @@ def dump(db, f, **options):
                               "signals": signals}
             frame_attributes = {}
             for frame_info in additionalFrameColums:  # Look for additional Frame Attributes
-                if hasattr(frame, frame_info):
-                    frame_attributes[frame_info] = getattr(frame, frame_info)
+                if frame.attribute(frame_info):  # does the attribute exist?
```

Comment:
This doesn't check it's existence.  If it exists but is `None` or anything else falsey it will be treated as not present.  Here's the first thing that came to my mind, though it is a touch smelly to me.  Maybe you can think of something nicer.

```python
MISSING = object()
value = frame.attribute(name=frame_info, default=MISSING)
if value is not MISSING:
    frame_attributes[frame_info] = value
```

Reply:
good point! As there is imho no reason to export None value, change to `is not None`